pkg/client/load_balancer.go

```diff
@@ -34,6 +34,26 @@ func (lb *LoadBalancerAPIService) CreateLoadBalancer(
 	return loadBalancerResp, err
 }
 
+func (lb *LoadBalancerAPIService) UpdateLoadBalancer(
+	ctx context.Context,
+	lbID int,
+	request models.CreateLoadBalancerRequest,
+) (models.CreateNetworkLoadBalancerResp, error) {
+	loadBalancerResp := models.CreateNetworkLoadBalancerResp{}
+	loadBalancerAPI := &api{
+		method: "PUT",
```

Comment:
Don't we need **compatibleVersion** here for the LB APIs ?

Reply:
Yes. It is not mandatory to add, but if the Load Balancer APIs are invoked  on setup such as 5.2.12, will there be no problem ?